chapter_appendix-mathematics-for-deep-learning/integral-calculus_vn.md

```diff
@@ -369,7 +368,7 @@ $$
 This tells us that the function changes by the area under a tiny sliver of a function.
 -->
 
-Nó cho ta biết hàm này thay đổi một khoảng bằng với diện tích phía dưới phần thay đổi cực nhỏ của hàm.
+Diện tích phía dưới phần thay đổi cực nhỏ của hàm cho ta biết hàm này có thay đổi.
```

Comment:
Câu này sửa lại hơi ngược với ý gốc;
```suggestion
Điều này cho ta biết hàm số có thay đổi bằng với diện tích phía dưới của một đoạn cực nhỏ của hàm.
```

Reply:
@lkhphuc câu này ý gốc là phần diện tích cho biết hàm thay đổi mà anh?
> This **tells** us that the function changes **by the area** under a tiny sliver of a function

Reply:
`the function changes by the area` 
`Changes by how much` là `thay đổi bao nhiêu` chứ không chỉ là có thay đổi. Mình hiểu ý ở đây là ta biết hàm này thay đổi đúng bằng với lượng diện tích dứoi đường cong bé xíu đó.

Reply:
` => f(x+e) = x + e` Cái này ở đâu ra vậy ông giáo. f(x) = x^2; f(1) = 1; f(1+0.1) = 1.21 /= 1.1

`This tells us` : Cái công thức ở ngay trên,
`that the function changes`: Vế trái F(x+ép) - F(x)
` by`: Dấu bằng
` the area`: Dấu tích phân,
` under a tiny sliver`: từ x tới x+eps
` of a function.`: f(y)dy

Reply:
em đang thử với 1 case đơn giản cụ thể, identity function `f(x) = x`, xem diện tích khi x thay đổi thành `x + e` (hàm này thì tính được diện tích bằng hình học đơn giản, ko cần tích phân) có bằng phần thay đổi của hàmko.
Chỗ `=> f(x+e) = x + e` là: với đầu vào `x+e` thì hàm này thành như vậy, đúng chứ? 
Phần tính change và area như trên.

Reply:
Mà chỗ 
`that the function changes: Vế trái F(x+ép) - F(x)`

`F` là nguyên hàm của `f` mà, function mình đang xét thay đổi là `f` chứ, đâu phải nguyên hàm của nó.

Reply:
Mình lost rồi ông giáo ạ, không hiểu 2 comments ở trên lắm.
F(x+\epsilon) - F(x) = \int_x^{x+\epsilon} f(y) \; dy.
$$
<!--
This tells us that the function changes by the area under a tiny sliver of a function.

F(x) là nguyên hàm của f(x) nhưng vẫn là hàm, f(x) là hàm đường cong, F(x) là hàm diện tích phía dưới của đường cong đó. 
Câu ở trên là họ từ `the function` đầu tiên là chỉ hàm F(x) vì từ trước vẫn đang nói về hàm diện tích, từ `a function` tiếp theo chỉ hàm `f(y)` vì họ vừa mới đổi thành một biến `y` mới.

Còn riêng câu dịch sửa lại là ` Diện tích phía dưới phần thay đổi cực nhỏ của hàm cho ta biết hàm này có thay đổi.` mình đọc không hiểu. Tại sao diện tích phía dưới là cho ta biết hàm có thay đổi, hàm constant f(x)=1 hay hàm identity f(x)=x thì đều có diện tích phía dưới F(x+eps) - F(x) thay đổi khi khoảng eps thay đổi mà.

Reply:
`This tells us that **the function (F)** changes by the area under a tiny sliver of **a function (f)**.`

ok em hiểu vấn đề ở đây rồi. Từ `function` đang bị dùng cho cả F và f, nên câu đề xuất phía dưới confusing:

`Điều này cho ta biết hàm số có thay đổi bằng với diện tích phía dưới của một đoạn cực nhỏ của hàm.`

TV ko có `a an the` nên nên dịch rõ ra. Phía trên em argue cho `f`, chứ ko xét `F` nên chả đi đến đâu 😄

```suggestion
Điều này cho ta biết hàm số $F(x)$ thay đổi bằng với diện tích phía dưới của một đoạn cực nhỏ của hàm $f(y)$.
```